src/object.d

```diff
@@ -2089,12 +2089,15 @@ auto byKeyValue(T : V[K], K, V)(T aa) pure nothrow @nogc
 {
     import core.internal.traits : substInout;
 
+    static auto safeAARangeEmpty(AARange rr) @safe { return () @trusted { return _aaRangeEmpty(rr); } (); }
```

Comment:
I'm not sure it's necessary to have these functions as they're only used once. I think an `@trusted` block should suffice.